vars/abapEnvironmentPipelineStagePrepareSystem.groovy

```diff
@@ -24,6 +26,7 @@ void call(Map parameters = [:]) {
         cloudFoundryCreateService script: parameters.script
         input message: "Steampunk system ready? Please make sure that you received the confirmation email before proceeding!"
         cloudFoundryCreateServiceKey script: parameters.script
+        abapEnvironmentCloneGitRepo script: parameters.script
```

Comment:
There is a stage called "Clone Repositories". What is the reason for including the step "abapEnvironmentCloneGitRepo" in the "Prepare System" stage?

Reply:
I removed it from the Prepare System Stage and added it to Clone Repositories Stage. Please check if this is as expected now.